frontend/src/main/pages/MyReviews/MyReviewsIndexPage.js

```diff
@@ -1,11 +1,45 @@
+import React from "react";
+import { useBackend } from "main/utils/useBackend";
+
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import ReviewTable from "main/components/Review/ReviewTable";
+import { Button } from "react-bootstrap";
+import { useCurrentUser, hasRole } from "main/utils/currentUser";
 
 export default function MyReviewsIndexPage() {
-  // Stryker disable all : placeholder for future implementation
+  const currentUser = useCurrentUser();
+
+  const createButton = () => {
+    if (hasRole(currentUser, "ROLE_ADMIN")) {
+      return (
+        <Button
+          variant="primary"
+          href="/reviews/create"
+          style={{ float: "right" }}
+        >
+          Create Reviews
+        </Button>
+      );
+    }
+  };
+
+  const {
+    data: reviews,
+    error: _error,
+    status: _status,
+  } = useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    ["/api/reviews"],
+    { method: "GET", url: "/api/reviews" },
+    [],
+  );
+
```

Comment:
As it stands right now /api/reviews takes a parameter.  So you either need to 
* (a) supply one (though I don't think this is the right approach)
* (b) change the backend so that it doesn't need one (this is probably the right approach).

Reply:
fixed! /api/reviews is no longer taking in a parameter